ICAFIX/hcp_fix_multi_run

```diff
@@ -771,6 +785,15 @@ log_Msg "Done renaming files"
 # Remove the 'fake-NIFTI' file created in fix_3_clean for high-pass filtering of the CIFTI (if it exists)
 $FSLDIR/bin/imrm ${concatfmrihp}.ica/Atlas
 
+#always delete things with too-generic names
+rm -f ${concatfmrihp}.ica/filtered_func_data.nii.gz ${concatfmrihp}.ica/Atlas.dtseries.nii
+
+#delete highpass intermediates
+if [[ "$DeleteIntermediates" == "TRUE" ]]
+then
+    rm -f ${concatfmri}.nii.gz ${concatfmrihp}.nii.gz ${concatfmri}_Atlas.dtseries.nii ${concatfmri}_Atlas_hp${hp}.dtseries.nii
```

Comment:
Ditto this comment https://github.com/Washington-University/HCPpipelines/pull/123/files#r300772789 that I had for this analogous section in ReApplyFixMultiRunPipeline.sh